### ISSUE TITLE
issue with setting up the development environment.

### ISSUE THREAD

**Issue by xiyuyi** [royerlab/aydin#193]
**Describe the bug**
Have errors running the test case after setting up the development environment following instructions described here: https://github.com/royerlab/aydin/blob/master/CONTRIBUTING.md

The following steps were successful (see screenshots below for details).
```
conda create -n aydin python=3.9
conda activate aydin
pip install -e .
pip install -r requirements/development.txt
```
Two problems were encountered after the installation (following the instructions):
1. Running the tests using `python -m pytest . --disable-pytest-warnings --show-capture=stderr` results in error reports. (Please see **tests errors** under "Additional context" below for details.).
3. Directly starting Aydin from the mac terminal by typing "aydin" resulted in errors. (Please see "Aydin starting up error" under "additional context" below for details) .

**To Reproduce**
To reproduce, one can follow the instructions to setup development environment in the contributing guidelines. (assuming the error is not system dependent).

**Expected behavior**
Get error messages described above and shown in "additional context" below.

**Desktop**
 - OS: macOS Monterey
 - Version: 12.3.1

**Additional context**
The following message shows that the installation is successful:
<img width="877" alt="Screen Shot 2022-06-12 at 4 53 17 PM" src="https://user-images.githubusercontent.com/8964244/173258741-2f637005-7ff3-4b97-8a8d-f2993c0980d4.png">

**tests errors**
Here are the error messages after running the tests:
<img width="877" alt="Screen Shot 2022-06-12 at 4 54 14 PM" src="https://user-images.githubusercontent.com/8964244/173258776-a55f22b7-cc62-4ebf-923f-99f3ad8a57c0.png">
<img width="877" alt="Screen Shot 2022-06-12 at 4 55 54 PM" src="https://user-images.githubusercontent.com/8964244/173258813-96336ff8-0b99-4a0c-83ae-40377822c5b4.png">
<img width="876" alt="Screen Shot 2022-06-12 at 4 56 18 PM" src="https://user-images.githubusercontent.com/8964244/173258830-fb2282ac-2608-4ccd-ab80-ae2020b1ec13.png">
<img width="879" alt="Screen Shot 2022-06-12 at 4 57 30 PM" src="https://user-images.githubusercontent.com/8964244/173258860-23b46370-f77f-41ed-bd5c-7b0dc85b4815.png">

**Aydin starting up error**
Here are the error messages after trying to start Aydin under the development environment:
<img width="877" alt="Screen Shot 2022-06-12 at 4 58 44 PM" src="https://user-images.githubusercontent.com/8964244/173258894-5b88831f-6ee3-4f25-af63-2e8397308883.png">
<img width="879" alt="Screen Shot 2022-06-12 at 4 59 42 PM" src="https://user-images.githubusercontent.com/8964244/173258914-b09a50c4-cbe8-48ae-b99f-37f4fe7deeda.png">

**Comment by AhmetCanSolak** [comment]
Thank you for the issue @xiyuyi . It seems you are missing `libomp`. We mention this here  in README.md: https://github.com/royerlab/aydin#mac-specific-dependencies=  and our documentation: https://royerlab.github.io/aydin/getting_started/install.html#install-aydin-in-a-conda-environment-with-cuda-support .  You  can get it with `brew install libomp`.  I will add this detail to CONTRIBUTING file too.

**Comment by AhmetCanSolak** [comment]
addressed in #194 .
